### ISSUE TITLE
[BUG] Crash on hidden Files

### ISSUE THREAD

**Issue by dunklesToast** [jonaswinkler/paperless-ng#1062]
**Describe the bug**
macOS creates a `._FILENAME` file if you open a directory with the Finder. This results in an Paperless import error which seems to crash the whole import process. It'll also create a `.DS_Store` file.

**To Reproduce**
Open your consume folder with the macOS Finder. It'll create a lot of unnecessary files (on an NFS Share).

**Expected behavior**
Since files beginning with a dot are hidden paperless-ng should ignore them while importing.

**Log**
```txt
21:38:46 [Q] INFO Process-1:1 processing [._MyPDF.pdf]
[2021-05-26 21:38:46,458] [INFO] [paperless.consumer] Consuming ._MyPDF.pdf
[2021-05-26 21:38:46,493] [ERROR] [paperless.consumer] Unsupported mime type application/octet-stream
21:38:46 [Q] INFO Process-1:1 stopped doing work
21:38:46 [Q] ERROR Failed [._MyPDF.pdf] - ._MyPDF.pdf: Unsupported mime type application/octet-stream : Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django_q/cluster.py", line 436, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 81, in consume_file
    task_id=task_id
  File "/usr/src/paperless/src/documents/consumer.py", line 211, in try_consume_file
    f"Unsupported mime type {mime_type}"
  File "/usr/src/paperless/src/documents/consumer.py", line 70, in _fail
    raise ConsumerError(f"{self.filename}: {log_message or message}")
documents.consumer.ConsumerError: ._MyPDF.pdf: Unsupported mime type application/octet-stream
```
**Relevant information**
 - Linux Debian 4.19.0-16-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19) x86_64 GNU/Linux
 - Paperless 1.4.0
 - Installation method: Docker with PostgreSQL

**Comment by jonaswinkler** [comment]
Fixed in 1.4.4

**Comment by dunklesToast** [comment]
Interesting. I've pulled latest docker images today but it's running 1.4.0. Have to check that out tomorrow but I guess we can close this. Thanks!
